src/utils/tools/serializer.ts

```diff
@@ -0,0 +1,17 @@
+import { ExportType } from 'contracts/types'
+const serializer = (payload: any, format: ExportType, customized: boolean = false) => {
```

Comment:
Sure we can it will look as following, the most of the properties will be optional for now, since the schema isn't consistent. 
```TS
interface Payload {
icons: string[],
exportAs?: ExportType,
exportSize?: number,
customizationConfig?: interfaceForThat,
}```
```
Maybe later in `v2` we can make it consistent.